flight/PiOS/STM32/pios_servo.c

```diff
@@ -868,7 +873,7 @@ void PIOS_Servo_Update(void)
 	// If some banks are oneshot and some are dshot (why would you do this?)
 	// get the oneshots firing first.
 	for (uint8_t i = 0; i < servo_cfg->num_channels; i++) {
-		if (channel_mask & (1 << 1)) {
+		if (channel_mask & (1 << i)) {
```

Comment:
This looks like a bugfix... but nothing in commit message about it?

Reply:
Fixed both, thanks :D